### DISCUSSION TITLE
Add birthday field to user profile data and include notification for followers

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#234] [Medium CIPs]
This was suggested by AgustinOlmedo and I think this is great idea. It's pretty simple, add one more field to user profile to input birthday and send notifications about coming birthday to followers. I believe this must be basic feature for social protocol.

cc @tijno 
![image](https://user-images.githubusercontent.com/20090030/155276350-f67798ac-ae94-44af-b3f4-e249d0392b25.png)

**Comment by ItsAditya-xyz** [comment]
lol @lazynina is already working on profile extra data which will solve all this

**Reply by brootle** [reply]
basic profile features like that should be stored as simple profile field, if you don't believe me, go ask my friend dharmesh
